requirements.txt

```diff
@@ -17,3 +17,4 @@ tensorflow
 texttable
 tweepy
 xgboost
+yulewalker
```

Comment:
Normally I am reluctant to add dependencies but let's go for it.

Reply:
I'd love to use scipy's function, but at the moment it's [just a stub](https://github.com/scipy/scipy/blob/ee9985aafef8d3d99e19b2594767cf5fb8bfc3e6/scipy/signal/_filter_design.py#L3621)